### PR TITLE
week3 & week4 tasks - konst

### PR DESCRIPTION
**Pull request by gavrilov**
week3 & week4 tasks

### CODE REVIEW

File: src/konst/week4/FrequencyOfCharacters.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+*/
+
+package konst.week4;
+
+public class FrequencyOfCharacters {
+    public static void main(String[] args) {
+        System.out.println(frequencyOfChars("AAABBCDD"));
+    }
+
+    public static String frequencyOfChars(String str) {
```
Comment (PendarShams) [inline src/konst/week4/FrequencyOfCharacters.java:13]: Is it better to consider that we might have lower and upper case?

File: src/konst/week3/ReverseNegativeNumbers.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        System.out.println(reverseNegative(-89));
+    }
+
+    public static int reverseNegative(int num) {
+        // Handle non-negative numbers
+        if (num >= 0) {
+            return num;
+        }
+
+        // Make the number positive for reversing
```
Comment (PendarShams) [inline src/konst/week3/ReverseNegativeNumbers.java:19]: it might be better to change the int to long to handle the overflow better!

**Review by PendarShams — CHANGES_REQUESTED** [review]
please check the comments

File: src/konst/week4/FrequencyOfCharacters.java (hunk 1 above)
Comment (PendarShams) [reply src/konst/week4/FrequencyOfCharacters.java:13]: ok thank you

**Review by PendarShams — APPROVED** [review]

**Review by Sandzhar98 — APPROVED** [review]
lgtm

**Pull request merged**